sigs.yaml

```diff
@@ -535,9 +535,9 @@ sigs:
   meetings:
   - description: Regular SIG Meeting
     day: Monday
-    time: "14:00"
-    tz: UTC
-    frequency: biweekly/triweekly
+    time: "16:00"
+    tz: Poland
```

Comment:
Just a quick question on this, do you know if  `Poland` will follow dst? Will it auto-switch between CET/CEST?